miasm2/ir/ssa.py

```diff
@@ -322,3 +322,20 @@ def reassemble_expr(self, e):
                 if id_rhs in self.expressions:
                     todo.add(id_rhs)
         return e
+
+
+class SSAPath(SSABlock):
```

Comment:
You may be right. Maybe the `analysis` module is more likely to receive such a great PR than `ir` itself.